spec/api/v1/trade_leads_spec.rb

```diff
@@ -62,9 +63,9 @@
         expect(json_response['offset']).to eq(0)
 
         results = json_response['results']
-        expect(results[0]).to eq(expected_results[1])
-        expect(results[1]).to eq(expected_results[2])
-        expect(results[2]).to eq(expected_results[3])
+        expect(results[0]).to eq(expected_results[4])
```

Comment:
For sure :-) In fact, I think we could move most of the detailed search testing into the Query specs, and just have these "api" (i.e. request) specs do a few sanity checks instead. Better to have all the edge case testing close to the implementation, and do fewer high-level tests.